src/enable_variable_ops/ngraph_rewrite_pass.cc

```diff
@@ -241,25 +241,58 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
     bool ngraph_not_enabled =
         (!config::IsEnabled()) || (std::getenv("NGRAPH_TF_DISABLE") != nullptr);
     bool already_processed = IsProcessedByNgraphPass(options.graph->get());
+    if (!already_processed && ngraph_not_enabled) {
+      NGRAPH_VLOG(0) << "Graph will not run through ngraph";
+    }
     if (ngraph_not_enabled || already_processed) {
-      NGRAPH_VLOG(1) << "Not running through nGraph. nGraph not enabled: "
-                     << ngraph_not_enabled
-                     << " Already processed: " << already_processed;
+      NGRAPH_VLOG(1) << std::string("Rewrite pass will not run because ") +
+                            (already_processed ? "graph is already preprocessed"
+                                               : "ngraph is disabled");
       NGraphClusterManager::EvictAllClusters();
       return Status::OK();
     }
 
+    // Get backend + its configurations
+    // to be attached to the nodes
+    // Precedence Order: Env Variable > BackendManager
+    std::unordered_map<std::string, std::string> config_map;
+    string backend_name = BackendManager::GetCurrentlySetBackendName();
+    const char* ng_backend_env_value = std::getenv("NGRAPH_TF_BACKEND");
+    if (ng_backend_env_value != nullptr) {
+      string backend_env = std::string(ng_backend_env_value);
+      if (backend_env.empty() ||
+          !BackendManager::IsSupportedBackend(backend_env)) {
+        return errors::Internal("NGRAPH_TF_BACKEND: ", backend_env,
+                                " is not supported");
+      }
+      backend_name = backend_env;
+      NGRAPH_VLOG(1) << "Setting backend from the enviornment variable "
+                        "NGRAPH_TF_BACKEND = "
+                     << backend_name;
+    } else {
+      NGRAPH_VLOG(1) << "Setting backend from the BackendManager ";
+    }
+
+    // splits into {"ngraph_backend", "_ngraph_device_config"}
+    config_map = BackendManager::GetBackendAttributes(
+        backend_name);  // SplitBackendConfig
+    backend_name = config_map.at("ngraph_backend");
+    config_map.erase("ngraph_backend");
+    NGRAPH_VLOG(0) << "NGraphEncapsulationPass: backend_name " << backend_name;
```

Comment:
This is VLOG(0) that means this will be received by data scientists etc. They don't need to know or should care about `NGraphEncapsulationPass:`. Change this to more user friendly message. For example: `nGraph using backend: CPU:0`